### ISSUE TITLE
CreateDistanceRequest in App

### ISSUE THREAD

**Issue by neil-marcellini** [Expensify/App#22711]
Please follow the plan in the design doc: [CreateDistanceRequest](https://docs.google.com/document/d/16Q9sE-991CZ4wUcalOs_U-qyEEza_C9tZwj6G5lHUp0/edit#heading=h.awqi6tozwmz5) in App

**Comment by neil-marcellini** [comment]
On HOLD, changing to a weekly while I wait.

**Comment by neil-marcellini** [comment]
On hold

**Comment by neil-marcellini** [comment]
Still holding

**Comment by tgolen** [comment]
We are taking this off hold and decided it can be built in parallel, just not hooked up to anything for now. I'll take this over and write up a PR.

**Comment by tgolen** [comment]
[The PR](https://github.com/Expensify/App/pull/24385) is created and is now in review.

**Comment by adelekennedy** [comment]
### Payouts due:

Contributor+: $500 @allroundexperts (NewDot) internal review

**Comment by adelekennedy** [comment]
Updated!

**Comment by tgolen** [comment]
Thank you, Jason! I'm going to close this out.
